### ISSUE TITLE
Colab Notebooks

### ISSUE THREAD

**Issue by bnmajor** [InsightSoftwareConsortium/itkwidgets#541]
Opening this issue to track some inconsistent and/or weird behaviors being seen in notebooks that are run in Colab. These issues are not seen when run with Jupyter Lab or SageMaker Studio Lab. All notebooks have been run with the "run all" option, last tested 9/27 with `v1.0a9`. These notebooks will need to be re-tested after #531 is completed so that they can be tested with the latest release.

- [x] [Hello3DWorld](https://github.com/InsightSoftwareConsortium/itkwidgets/blob/main/examples/Hello3DWorld.ipynb): Almost always needs the `view` cell to be run twice. Sometimes works on the first run, always works on the second.
- [x] [DaskArray](https://github.com/InsightSoftwareConsortium/itkwidgets/blob/main/examples/integrations/dask/DaskArray.ipynb): First call to `view` must be called twice, second call to `view` never works no matter how many times it is re-run (except for 1 out of 10 tries where it worked the first time).
- [x] [transform_visualization](https://github.com/InsightSoftwareConsortium/itkwidgets/blob/main/examples/integrations/MONAI/transform_visualization.ipynb): Does not work yet because of data size, relies on #531 in order to use the latest itkwidgets release.
- [x] [DataArray](https://github.com/InsightSoftwareConsortium/itkwidgets/blob/main/examples/integrations/xarray/DataArray.ipynb): First `view` cell always needs to be run twice. Second `view` cell rarely but occasionally works the first time, always the second time.

**Comment by thewtex** [comment]
@bnmajor done via #557 ?

**Comment by bnmajor** [comment]
@thewtex Yes, I'd say so! I did find that there is an issue around notebooks with multiple `view` calls when you use `Run All` where the first invocation never displays anything. This is *not* exclusively a Colab propblem though so I will open a new issue for that.
